library/src/scala/tasty/reflect/SourceCodePrinter.scala

```diff
@@ -5,7 +5,7 @@ import scala.annotation.switch
 import scala.quoted.show.SyntaxHighlight
 
 /** Printer for fully elaborated representation of the source code */
-class SourceCodePrinter[R <: Reflection & Singleton](val reflect: R)(syntaxHighlight: SyntaxHighlight) extends Printer[R] {
+class SourceCodePrinter[R <: Reflection & Singleton](val reflect: R)(syntaxHighlight: SyntaxHighlight, printTypePrefix: Boolean = true) extends Printer[R] {
```

Comment:
I will move things around in #10162, we would probably need to receive this parameter in a slightly different place.
I would also remove the default value and add it explicitly at the call site.

Reply:
I would rename `printTypePrefix` to `fullNames`

Reply:
Ok, I just most likely wait for your PR to be merged and then work a bit more on the mine.

Reply:
@tgodzik now you can rebase.

Reply:
I finally rebased and renamed the boolean flag.